docs/source/01_intro.md

```diff
@@ -1,38 +1,48 @@
-* Join our mailing list at [http://bdgenomics.org/mail](http://bdgenomics.org/mail)
-* Checkout the current build status at [https://amplab.cs.berkeley.edu/jenkins/](https://amplab.cs.berkeley.edu/jenkins/view/Big%20Data%20Genomics/)
-* Download official releases at [https://github.com/bigdatagenomics/adam/releases](https://github.com/bigdatagenomics/adam/releases)
-* View our software artifacts on Maven Central at [http://search.maven.org/#search%7Cga%7C1%7Corg.bdgenomics](http://search.maven.org/#search%7Cga%7C1%7Corg.bdgenomics)
-* See our snapshots at [https://oss.sonatype.org/index.html#nexus-search;quick~bdgenomics](https://oss.sonatype.org/index.html#nexus-search;quick~bdgenomics)
-* Look at our CHANGES file at [https://github.com/bigdatagenomics/adam/blob/master/CHANGES.md](https://github.com/bigdatagenomics/adam/blob/master/CHANGES.md)
-
-ADAM is a genomics analysis platform with specialized file formats built using [Apache Avro](http://avro.apache.org), [Apache Spark](http://spark.apache.org/) and [Parquet](http://parquet.io/). Apache 2 licensed.  
+* Follow our Twitter account at
+[https://twitter.com/bigdatagenomics/](https://twitter.com/bigdatagenomics/)
+* Chat with ADAM developers at
+[https://gitter.im/bigdatagenomics/adam](https://gitter.im/bigdatagenomics/adam)
+* Join our mailing list at
+[http://bdgenomics.org/mail](http://bdgenomics.org/mail)
+* Checkout the current build status at
+[https://amplab.cs.berkeley.edu/jenkins/](https://amplab.cs.berkeley.edu/jenkins/view/Big%20Data%20Genomics/)
+* Download official releases at
+[https://github.com/bigdatagenomics/adam/releases](https://github.com/bigdatagenomics/adam/releases)
+* View our software artifacts on Maven Central at
+[http://search.maven.org/#search%7Cga%7C1%7Corg.bdgenomics](http://search.maven.org/#search%7Cga%7C1%7Corg.bdgenomics)
+* See our snapshots at
+[https://oss.sonatype.org/index.html#nexus-search;quick~bdgenomics](https://oss.sonatype.org/index.html#nexus-search;quick~bdgenomics)
+* Look at our CHANGES file at
+[https://github.com/bigdatagenomics/adam/blob/master/CHANGES.md](https://github.com/bigdatagenomics/adam/blob/master/CHANGES.md)
+
+ADAM is a genomics analysis platform with specialized file formats built using
+[Apache Avro](http://avro.apache.org), [Apache Spark](http://spark.apache.org/)
+and [Parquet](http://parquet.io/). Apache 2 licensed.
 
 ## Apache Spark
 
-[Apache Spark](http://spark.apache.org/) allows developers to write algorithms in succinct code that can run fast locally, on an in-house cluster or on Amazon, Google or Microsoft clouds. 
+[Apache Spark](http://spark.apache.org/) allows developers to write algorithms
+in succinct code that can run fast locally, on an in-house cluster or on
+Amazon, Google or Microsoft clouds.
 
-For example, the following code snippet will print the top 10 21-mers in `NA2114` from 1000 Genomes.
+For example, the following code snippet will print the top 10 21-mers in
+`NA2114` from 1000 Genomes.
```

Comment:
Are we not just closing this PR? Or are we merging this with #1653? I only ask because your review comment suggested we don't merge.